components/Gallery/index.js

```diff
@@ -19,7 +19,11 @@ export default function Gallery({ onCartItem }) {
     <Styled.GallerySection>
       <Styled.GalleryList>
         {isLoading ? (
-          <p>Is Loading...</p>
+          <>
+            <br />
+            <Styled.LoadingCube />
+            <br />
+          </>
```

Comment:
```suggestion

            <Styled.LoadingCube />

```

Reply:
This one is on my to-do will refactor the LoadingCube in the next story.
There are some edge case visual bugs I had to fix.
I already got a solution for it.

Reply:
- [x] Fixed with a margin for now.